site/public/mjs/sql-toolbox.js

```diff
@@ -73,3 +73,9 @@ async function runSqlQuery() {
         console.error(exc);
```

Comment:
It would be better to notify the user (even with `alert(exc)`)  as network/serialization failure is caught here.